src/node/desktop/binding.gyp

```diff
@@ -0,0 +1,14 @@
+{
+  "targets": [
+    {
+      "target_name": "macos",
+      "cflags!": [ "-fno-exceptions" ],
+      "cflags_cc!": [ "-fno-exceptions" ],
+      "sources": [ "src/native/macos/hello.cc" ],
```

Comment:
It would be worth discussing if this structure makes sense -- that is, we place our native bindings in `src/native`, with platform-specific sub-directories as appropriate.

Reply:
https://github.com/node-ffi/node-ffi/blob/d9b843c03b35b99f205d001ebcbedd1b6c63b3d0/binding.gyp#L16-L21 for an example of platform-specific sources.